xlive/Blam/Engine/interface/new_hud_draw.cpp

```diff
@@ -747,6 +758,32 @@ void __cdecl draw_hud_player_indicators(uint32 local_render_user_index)
 }
 /* public code */
 
+void hud_draw_on_map_load()
+{
+	if (g_draw_hud_crosshair_bitmap_cache)
+		free(g_draw_hud_crosshair_bitmap_cache);
```

Comment:
g_draw_hud_crosshair_bitmap_cache should also be freed when we close the game, not just here